sktime/transformations/series/tests/test_theta.py

```diff
@@ -6,28 +6,41 @@
 __all__ = []
 
 import numpy as np
+import pytest
 from scipy.stats import linregress
 
 from sktime.datasets import load_airline
 from sktime.transformations.series.theta import ThetaLinesTransformer
 
 
 def test_theta_0():
+    # with theta = 0
     y = load_airline()
     t = ThetaLinesTransformer(0)
     t.fit(y)
-    actual = t.transform(y)
+    line = t.transform(y)
+    actual = line[:, 0]
     x = np.arange(y.size) + 1
     lin_regress = linregress(x, y)
-    expected_0 = lin_regress.intercept + lin_regress.slope * x
+    expected = lin_regress.intercept + lin_regress.slope * x
 
-    np.testing.assert_array_equal(actual, expected_0)
+    np.testing.assert_almost_equal(actual, expected, decimal=8)
 
 
 def test_theta_1():
+    # with theta = 1 Theta-line is equal to the original time-series
     y = load_airline()
     t = ThetaLinesTransformer(1)
     t.fit(y)
-    actual = t.transform(y)
-
+    line = t.transform(y)
+    actual = line[:, 0]
```

Comment:
this shouldn't be necessary anymore now that we return pd.Series for a single theta